packages/service/src/utils/users.ts

```diff
@@ -1,5 +1,26 @@
+import { upsert } from "zapatos/db";
 import { db, pool, schema } from "../db";
 
 export function createUser(props: { name: string }) {
-  return db.upsert("users", props, "name").run(pool);
+  // return db.upsert("users", props, "name").run(pool);
+  return db.sql<
+    schema.users.SQL,
+    schema.users.Selectable
+  >`INSERT INTO ${"users"} (${"name"}) VALUES (${{ name: props.name }})`.run(
+    pool
+  );
 }
+
+export const selectUsers = () => {
+  const users = db.sql<
+    schema.users.SQL,
+    schema.users.Selectable[]
+  >`SELECT * FROM ${"users"}`.run(pool);
+  return users;
+};
+
+export const createPswash = (user: { password: string; id: string }) => {
```

Comment:
Couple nits here
* I expect `create` functions to be `INSERT` transactions not `UPDATES`
* `Pswash` is a typo, I think, for `Pswhash` but I don't like that name either. 

This should probably be called `updatePassword` and I would expect it take the current password, and the new password, and the user id. We should match the current password before updating.